bundles/automation/org.eclipse.smarthome.automation.api/META-INF/MANIFEST.MF

```diff
@@ -8,12 +8,20 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Import-Package: 
  org.eclipse.smarthome.config.core,
  org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.storage,
+ org.json,
```

Comment:
This bundle must not have a dependency to org.json, only the parser.json should have that dependency!